src/Mod/Part/App/TopoShapeFacePyImp.cpp

```diff
@@ -878,6 +878,8 @@ PyObject* TopoShapeFacePy::cutHoles(PyObject *args)
 Py::Object TopoShapeFacePy::getSurface() const
 {
     const TopoDS_Face& f = TopoDS::Face(getTopoShapePtr()->getShape());
+    if (f.IsNull())
+        return Py::Object(Py_None);

```

Comment:
Here you would normally use `return Py::None()`

Reply:
OK, thanks